Formula/e/envconsul.rb

```diff
@@ -20,6 +20,7 @@ class Envconsul < Formula
   end
 
   deprecate! date: "2024-02-05", because: "depends on soon to be deprecated consul"
+  disable! date: "2025-02-08", because: "depends on soon to be disabled consul"
```

Comment:
I guess it's not strictly disallowed (maybe it should be?), but typically the `disable` block replaces the `deprecate` block

```suggestion
  disable! date: "2025-02-08", because: "depends on soon to be disabled consul"
```

Reply:
Maybe it hasn't changed yet, but I thought we were moving toward keeping both for historical purposes in `brew info`.

Reply:
Ah I wasn't aware, I guess that change is https://github.com/Homebrew/brew/pull/18661